src/Component/AuthPage.jsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Form } from "react-bootstrap"
+import { Navigate, useNavigate } from "react-router-dom";
+import { auth } from "../firebase";
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword } from "firebase/auth";
+import { useEffect, useState } from "react";
+
+export default function AuthPage(props){
+  let {
+    isAuth,
+    setIsAuth,
+    setUser,
+    user,
+  } = props
+
+  useEffect(()=>{
+    console.log('user: ', user)
+    console.log("display name: ", user.displayName)
+  },[])
+
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+
+  const navigate = useNavigate()
+
+  const signUp = () =>{
+    createUserWithEmailAndPassword(auth, email, password)
+    .then((userCredential) =>{
+      setIsAuth(true)
+      setUser(userCredential.user);
+      navigate("/")
+    })
+    .catch((err) => console.error(err))
+  }
+
+  const signIn = () => {
+    signInWithEmailAndPassword(auth, email, password)
+    .then((userCredential) => {
+      setIsAuth(true);
+      setUser(userCredential.user)
+      navigate('/')
+    })
+    .catch((err) => console.error(err))
+  }
+
+  return (
```

Comment:
Great use of auto focus in your JSX nice touch/ nice use of font awesome!
Note that breaks are conventionally self closing like images and inputs - <br />